wavefront_opentracing_sdk/tracer.py

```diff
@@ -26,12 +47,26 @@ def __init__(self, reporter, application_tags, global_tags=None):
         :type application_tags: :class:`ApplicationTags`
         :param global_tags: Global tags for the tracer
         :type global_tags: list of pair
+        :param samplers: Samplers for the tracer
+        :type samplers: list of samplers
         """
         super(WavefrontTracer, self).__init__(ThreadLocalScopeManager())
         self._reporter = reporter
         self._tags = global_tags or []
         self._tags.extend(application_tags.get_as_list())
+        self._samplers = samplers
         self.registry = registry.PropagatorRegistry()
+        self.application_service_prefix = "%s.%s." % (
```

Comment:
Please use `.format()` function when constructing strings instead of C style `%` formatting

Reply:
Please note this rule doesn't apply to string templates passed to logger